Source/SpacePark/DatabaseModels/SpaceShip.cs

```diff
@@ -8,9 +8,21 @@ public class SpaceShip
 
         public string Name { get; set; }
 
-        public int ShipLength { get; set; }
-
-        [NotMapped]
         public string Length { get; set; }
+
+        public Person PersonID { get; set; }
+
+        public static SpaceShip CreateStarshipFromAPI(string url)
```

Comment:
Why is GreateStarshipFromAPI returning a SpaceShip?? ;) What should I make of this?